### PR TITLE
[CP-498] Implement UI part

### PR DESCRIPTION
**Pull request by igorbezsmertnyi**
Jira: [CP-498]

**Description**
UI implementation

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-498]: https://appnroll.atlassian.net/browse/CP-498?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by wercix — DISMISSED** [review]

File: packages/app/src/files-manager/components/files-manager-panel/files-manager-panel.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/**
```
Comment (wercix) [inline packages/app/src/files-manager/components/files-manager-panel/files-manager-panel.styles.tsx:1]: The name of this file should be `styled`, not `styles` like another files with style 😉

Comment (igorbezsmertnyi) [reply packages/app/src/files-manager/components/files-manager-panel/files-manager-panel.styles.tsx]: https://github.com/mudita/mudita-center/pull/905/commits/f5ec35d0de138b5fc78080461cc5e4b24599e745

Comment (wercix) [reply packages/app/src/files-manager/components/files-manager-panel/files-manager-panel.styles.tsx]: Good, but remove additional space before `styled` 🙈

File: packages/app/src/files-manager/components/files-manager-panel/files-manager-panel.component.test.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+  return renderWithThemeAndIntl(
+    <Provider store={store}>
+      <FilesManagerPanel {...props} />
+    </Provider>
+  )
+}
+
+describe("When Mudita Harmony connected", () => {
+  test("don't render component component", () => {
```
Comment (wojcik92michal) [inline packages/app/src/files-manager/components/files-manager-panel/files-manager-panel.component.test.tsx:45]:
```suggestion
  test("don't render component", () => {
```

File: packages/app/src/files-manager/components/files-manager-panel/files-manager-panel.component.test.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    } as ReduxRootState)
+
+    expect(
+      queryByTestId(FilesManagerPanelTestIds.Wrapper)
+    ).not.toBeInTheDocument()
+  })
+})
+
+describe("When Mudita Pure connected", () => {
+  test("renders component component", () => {
```
Comment (wojcik92michal) [inline packages/app/src/files-manager/components/files-manager-panel/files-manager-panel.component.test.tsx:59]:
```suggestion
  test("renders component", () => {
```

**Review by wojcik92michal — APPROVED** [review]

**Review by wercix — APPROVED** [review]

**Pull request merged**